django/test/client.py

```diff
@@ -402,6 +402,7 @@ def generic(self, method, path, data='',
         parsed = urlparse(str(path))  # path can be lazy
         data = force_bytes(data, settings.DEFAULT_CHARSET)
         r = {
+            'HTTP_ACCEPT_LANGUAGE': os.environ.get('LANG', '*'),
```

Comment:
The [`Accept-Language` format](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4) and `LANG` environment variable also don't share the same format.

For example, I've got `LANG=fr_CA.UTF-8` defined locally while `Accept-Language` should be `fr-ca`.

Reply:
FTR LANG=fr-FR and LANG=fr_FR or LANG=fr works equally well for Firefox although that may be breaking the expected format.